REVIEW_CHECKLIST.md

```diff
@@ -0,0 +1,35 @@
+# Review Checklist
+
+When reviewing, keep in mind that we follow [Ansible Code of Conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html) in all our contributions and interactions within this repository.
+
+**General tips**
+- Try to create a culture of collaboration when reviewing
+- Welcome the author and thank them for the pull request
+- When suggesting changes, try to use questions, not statements
+- When suggesting mandatory changes, do it as politely as possible providing documentation references
+- If your suggestion is optional or a matter of personal preferences, please say it explicitly
+- When asking for adding tests or for complex code refactoring, say that the author is welcome to ask for clarifications and help if they need
+- If somebody suggests a good idea, say about it or put a thumbs up
```

Comment:
Please feel free to ignore this as I'm really not sure about this myself.
```suggestion
- If somebody suggests a good idea, mention it or put a thumbs up
```

Reply:
sounds better, thanks! committed